charts/clusterapi-resources/templates/VSphereMachineTemplate.yaml

```diff
@@ -27,6 +27,10 @@ spec:
     {{- if and $t.spec.tagIDs (gt (len $t.spec.tagIDs) 0)  }}
       tagIDs:
       {{- toYaml ($t.spec.tagIDs | default $defaultSpec.tagIDs) | nindent 6 }}
+    {{- end }}
+    {{- if and $t.spec.pciDevices (gt (len $t.spec.pciDevices) 0)  }}
+      pciDevices:
+      {{- toYaml ($t.spec.pciDevices | default $defaultSpec.pciDevices) | nindent 6 }}
```

Comment:
Should we use EITHER the default devices or the list of passed? Or should they be merged (the defaults + the list passed)?

Reply:
Merges are hard to handle. I chose to use the override option, since is the most common behavior for helm values.

Reply:
I agree, managing lists on Helm suck.